restangular/restangular.d.ts

```diff
@@ -116,6 +116,9 @@ declare namespace restangular {
     withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
     save(queryParams?: any, headers?: any): IPromise<any>;
     getRestangularUrl(): string;
+    route?: string;
+    id?: string;
```

Comment:
Yeah that would be a better approach, I don't see a case that it would be
an object.

On Wed, May 25, 2016, 11:55 AM Blaz <notifications@github.com> wrote:

> In restangular/restangular.d.ts
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9240#discussion_r64597763>
> :
>
> > @@ -116,6 +116,9 @@ declare namespace restangular {
> >      withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
> >      save(queryParams?: any, headers?: any): IPromise<any>;
> >      getRestangularUrl(): string;
> > +    route?: string;
> > +    id?: string;
>
> Maybe using type of any would be
> better here to accommodate more endpoints?
>
> If it can be an object then I guess any is fine, otherwise we can use
> string or number id?: string | number;
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9240/files/83dcde9de2c6259aeafab1c0c68929bea0f95a2d#r64597763>
>